pkgs/stdenv/linux/bootstrap-files/x86_64.nix

```diff
@@ -1,9 +1,12 @@
-# Use busybox for i686-linux since it works on x86_64-linux as well.
-(import ./i686.nix) //
-
 {
+  busybox = import <nix/fetchurl.nix> {
+    url = https://lblasc-nix-dev.s3-eu-west-1.amazonaws.com/0zaxybl83qf0qc8bd3qb6z084jx94xv0-stdenv-bootstrap-tools/on-server/busybox;
+    sha256 = "1jyhi0irfgd8c8hwck3ym2qn8v29s3jm6a3cfilvilxjcblhzngx";
+    executable = true;
+  };
+
   bootstrapTools = import <nix/fetchurl.nix> {
-    url = http://tarballs.nixos.org/stdenv-linux/x86_64/4907fc9e8d0d82b28b3c56e3a478a2882f1d700f/bootstrap-tools.tar.xz;
```

Comment:
Why do these need to be updated? While we have to do this regularly for darwin compatibility, there hasn't been a reason to update these for linux since early 2016.

Reply:
Minimal gcc version for building glibc is now 6.2, bootstrap tools provide only version 5. Also some other libs are out of date.

Reply:
Hmm, that _could_ be worked around by reorganizing the bootstrapping process, but I'd probably choose the easier route of updating the bootstrap-tools.